tasks/main.yml

```diff
@@ -16,6 +16,8 @@
 - set_fact:
     _force_tls: False
     _use_tls: False
+    _vhost_conffile_prefix: "/etc/httpd/conf.d/{{ _website_domain }}"
+    _vhost_confdir: "/etc/httpd/conf.d/{{ _website_domain }}.conf.d"
 
```

Comment:
I wonder if we shouldn't get _vhost_conffile_prefix be used. But that's a cleanup (and i am not sure if it work)